src/devices/tuya.ts

```diff
@@ -8712,6 +8701,420 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0726', ['_TZ3000_kt6xxa4o']),
+        model: 'QAT42Z3',
+        vendor: 'QA',
+        description: 'QA 3CH Scene Switch',
```

Comment:
For all `description`

- Remove `QA` as it's already in `vendor`
- Make it lower case, except the first letter
- `CH` -> `channel`